pages/store.php

```diff
@@ -275,16 +275,17 @@ function store() {
 		<b>Your money:</b> &yen;{$player->getMoney()}</td></tr></table>
 
 		<p style='text-align:center;margin-bottom:0;'>
-			<a href='$self_link&view=jutsu&jutsu_type=ninjutsu' " . ($jutsu_type == 'ninjutsu' ? $style : "") . ">Ninjutsu</a> |
-			<a href='$self_link&view=jutsu&jutsu_type=taijutsu' " . ($jutsu_type == 'taijutsu' ? $style : "") . ">Taijutsu</a> |
-			<a href='$self_link&view=jutsu&jutsu_type=genjutsu' " . ($jutsu_type == 'genjutsu' ? $style : "") . ">Genjutsu</a>
+			<a href='$self_link&view=jutsu&jutsu_type=ninjutsu' " . ($jutsu_type == 'ninjutsu' ? $style : "") . " style='text-decoration:none'>Ninjutsu</a> |
+			<a href='$self_link&view=jutsu&jutsu_type=taijutsu' " . ($jutsu_type == 'taijutsu' ? $style : "") . " style='text-decoration:none'>Taijutsu</a> |
+			<a href='$self_link&view=jutsu&jutsu_type=genjutsu' " . ($jutsu_type == 'genjutsu' ? $style : "") . " style='text-decoration:none'>Genjutsu</a>
```

Comment:
```suggestion
			<a href='$self_link&view=jutsu&jutsu_type=ninjutsu' class='" . ($jutsu_type == 'ninjutsu' ? 'selected' : "") . "'>Ninjutsu</a> |
			<a href='$self_link&view=jutsu&jutsu_type=taijutsu' class='" . ($jutsu_type == 'taijutsu' ? 'selected' : "") . "'>Taijutsu</a> |
			<a href='$self_link&view=jutsu&jutsu_type=genjutsu' class='" . ($jutsu_type == 'genjutsu' ? 'selected' : "") . "'>Genjutsu</a>
```

Reply:
Let's switch this to class-based and then address this in the layout style files for all links 🙏